src/microbe_stage/TweakedProcess.cs

```diff
@@ -50,7 +50,8 @@ public bool Equals(TweakedProcess other)
             return false;
 
         // ReSharper disable once CompareOfFloatsByEqualityOperator
-        return Rate == other.Rate && ReferenceEquals(Process, other.Process);
+        return Rate == other.Rate && SpeedMultiplier == other.SpeedMultiplier
```

Comment:
Adding comparing the speed multiplier here, I think might break some grouping of process displays...

So I would remove this change.

Reply:
I don't think it's right, because if processes with different states are considered equal, then they aren't correctly updated on GUI, unless I'm missing something.

Reply:
Have you playtested and confirmed that is a bug? Kind of unhelpfully the process list detection uses equality to combine items, so too strict checking can result in a situation where duplicates get created into the process list.

Reply:
It's true that if this isn't the type of equality used in the GUI combining (and that doesn't call this equality in turn), this should be fine. Though, the last time I tried to improve the process equality checks to be more accurate, I totally messed up the process speed panel to show each instance of a process separately (instead of keeping the functionality where all organelles are summed into one process of each type). I'll need to check this carefully when doing the next review (after the extra temporary memory use is solved).